### PR TITLE
Feature add products api reference

### PR DESCRIPTION
**Pull request by delagroove**

### CODE REVIEW

File: docs/products.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
+## Overview 
+Products is one of the most used entities in Mailchimp Open Commerce. It is basically what you offer in the store to sell, it can be tangible or intangible, it can also represent a physical product or service.
+
+Products can have 2 levels of variants, for example, a t-shirt can be sold by size and color.
```
Comment (MohanNarayana) [inline docs/products.md]: We can talk about 'options' here. https://github.com/reactioncommerce/reaction-docs/blob/trunk/public-docs/concepts-products.md

Comment (delagroove) [reply docs/products.md]: I'll link to this concept.

File: docs/products.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+-  `variants`: All variants of the product. Type `ProductVariant`
+-  `pricing`: Pricing for the product. Type `ProductPricingInfo`
+
+### Creating products
+
+in order to create products we need to call the mutation `createProduct` to create a base product with no variants. the parameter `shouldCreateFirstVariant`  in the `CreateProductInput` field can help create a first placeholder variant with the product creation.
+
+### Creating Variants
+
+Take a look at the `createProductVariant` mutation. by defining a `CreateProductVariantInput`, which is comprised of a productID and a shopID.
```
Comment (MohanNarayana) [inline docs/products.md]: I think there is a punctuation correction required here.

File: docs/products.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+-  `shop`: the related Shop for this product: Type `Shop`
+-  `socialMetadata`: metadata for social networks. Type `SocialMetadata`.
+-  `supportedFulfillmentTypes`: the types of fulfillment the product supports. Type `FulfillmentType`.
+-  `Tags`: Tags relalted to the product. Type `TagsConnection`.
+-  `variants`: All variants of the product. Type `ProductVariant`
+-  `pricing`: Pricing for the product. Type `ProductPricingInfo`
+
+### Creating products
+
+in order to create products we need to call the mutation `createProduct` to create a base product with no variants. the parameter `shouldCreateFirstVariant`  in the `CreateProductInput` field can help create a first placeholder variant with the product creation.
```
Comment (MohanNarayana) [inline docs/products.md]: Capitalize start of the sentence and headers.

**Review by MohanNarayana — CHANGES_REQUESTED** [review]

File: docs/products.md (hunk 2 above)
Comment (delagroove) [reply docs/products.md]: changed the sense of the phrase

File: docs/products.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+### Creating Variants
+
+To create variants take a look at the `createProductVariant` mutation. You must need to define a `CreateProductVariantInput`, comprised of a productID and a shopID.
+
+### Create, Clone & Archive
+
+Most of the create mutations also have a clone and an archive equivalent, to allow the fast creation of full CRUD-based operations. For example, `cloneProducts`,  `CloneProductVariants` and `archiveProducts` are available.
+
+
+#### mutation CreateProduct fields
```
Comment (MohanNarayana) [inline docs/products.md]: All heading to begin with capitalized letters

**Pull request closed without merging**